### PR TITLE
Documentation improvement, extra log output and welcome message for empty Tobiras

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
See commit messages. The welcome message looks like this:

image

### CODE REVIEW

File: frontend/src/i18n/locales/de.yaml [hunk 1]
```diff
@@ -18,7 +18,14 @@ general:
...
     Seite verlassen? Vorgenommene Änderungen sind möglicherweise noch nicht gespeichert!
   logo-alt: Das Logo von „{{title}}“
+  no-root-children: Noch keine Seiten...
 
+welcome:
+  title: Willkommen zu Tobira!
+  body: >
+    Die Hauptseite scheint noch leer zu sein! Um Inhalte oder weitere Seiten
+    hinzuzufügen, melden Sie sich zuerst mit einem Account (mit Moderator-Rechten)
+    an. Danach finden Sie auf der linken Seite Buttons zum Editieren von Seiten.
```
Comment (JulianKniephoff) [inline frontend/src/i18n/locales/de.yaml:28]: Not 100% sure but maybe

```suggestion
    hinzuzufügen, melden Sie sich zuerst mit einem Konto (mit Moderator-Rechten)
    an. Danach finden Sie auf der linken Seite Schaltflächen zum Editieren von Seiten.
```

Or something like that? :thinking: Could also make it `Zugang` and `Bedienelemente` ...

File: frontend/src/i18n/locales/de.yaml [hunk 2]
```diff
@@ -18,7 +18,14 @@ general:
   leave-page-confirmation: >
     Seite verlassen? Vorgenommene Änderungen sind möglicherweise noch nicht gespeichert!
   logo-alt: Das Logo von „{{title}}“
+  no-root-children: Noch keine Seiten...
 
+welcome:
+  title: Willkommen zu Tobira!
+  body: >
+    Die Hauptseite scheint noch leer zu sein! Um Inhalte oder weitere Seiten
+    hinzuzufügen, melden Sie sich zuerst mit einem Account (mit Moderator-Rechten)
```
Comment (JulianKniephoff) [inline frontend/src/i18n/locales/de.yaml:27]: I would would remove the parentheses here, like in the English version. It's essential information, after all.

File: frontend/src/i18n/locales/de.yaml [hunk 3]
```diff
@@ -18,7 +18,14 @@ general:
   leave-page-confirmation: >
     Seite verlassen? Vorgenommene Änderungen sind möglicherweise noch nicht gespeichert!
   logo-alt: Das Logo von „{{title}}“
+  no-root-children: Noch keine Seiten...
```
Comment (JulianKniephoff) [inline frontend/src/i18n/locales/de.yaml:21]: Not sure I like the ellipsis. I personally would put a space in front of it, but also in general, it always looks kind of passive aggressive to me. :shrug:

File: frontend/src/ui/Card.tsx [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import { FiAlertTriangle, FiInfo } from "react-icons/fi";
+import { FiAlertTriangle, FiInfo, FiSunrise } from "react-icons/fi";
 import { match } from "../util";
 
 
 type Props = JSX.IntrinsicElements["div"] & {
-    kind: "error" | "info";
+    kind: "error" | "info" | "welcome";
```
Comment (JulianKniephoff) [inline frontend/src/ui/Card.tsx:6]: This seems a bit specific for this rather generic component. :thinking: :shrug:

File: backend/src/http/handlers.rs [hunk 5]
```diff
@@ -248,10 +248,10 @@ impl CommonHeadersExt for hyper::http::response::Builder {
         //
         // - `style-src` has to include `unsafe-inline` due to Paella player
         //   emitting inline CSS. Our CSS lib, emotion-js, uses inline CSS, but
-        //   it allows passing it a nonce. We already use that nonce system,
-        //   which is kinda useless while we still have `unsafe-inline` due to
-        //   Paella anyway. But it means once Paella is changed to work in
-        //   strict CSP environment, it's easier to remove. Paella issue:
+        //   it allows passing it a nonce. Everything is prepared to use that
+        //   `nonce` system, but we can't include 'nonce-123' in the CSP header
+        //   as then browsers will diregard the 'unsafe-inline'. We hope to
```
Comment (JulianKniephoff) [inline backend/src/http/handlers.rs:253]:
```suggestion
        //   as then browsers will disregard the 'unsafe-inline'. We hope to
```

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Various comment.

File: frontend/src/ui/Card.tsx (hunk 4 above)
Comment (LukasKalbertodt) [reply frontend/src/ui/Card.tsx:6]: Absolutely, had the same thought. But couldn't come up with a solution that's not vastly more work. Any idea?
But also, I don't super mind.

File: frontend/src/i18n/locales/de.yaml (hunk 3 above)
Comment (LukasKalbertodt) [reply frontend/src/i18n/locales/de.yaml:21]:
```suggestion
  no-root-children: Noch keine Seiten ...
```

File: frontend/src/i18n/locales/de.yaml (hunk 2 above)
Comment (LukasKalbertodt) [reply frontend/src/i18n/locales/de.yaml:27]:
```suggestion
    hinzuzufügen, melden Sie sich zuerst mit einem Account mit Moderator-Rechten
```

File: frontend/src/i18n/locales/en.yaml [hunk 6]
```diff
@@ -17,6 +17,15 @@ general:
   homepage: Homepage
   leave-page-confirmation: Leave page? Changes you made may not be saved!
   logo-alt: Logo of “{{title}}”
+  no-root-children: No pages yet...
```
Comment (LukasKalbertodt) [inline frontend/src/i18n/locales/en.yaml:20]:
```suggestion
  no-root-children: No pages yet ...
```

File: frontend/src/ui/Card.tsx (hunk 4 above)
Comment (JulianKniephoff) [reply frontend/src/ui/Card.tsx:6]: Dunno if this qualifies as vastly more work already, but one idea I had was to extract all of the decisions that are made based on this enum into a data structure which you could pass in. That would contain sth like the icon used, background color, text color, ... (or for simplicity's sake just a whole CSS thing) :shrug:

**Review by JulianKniephoff — APPROVED** [review]

**Pull request merged**